### PR TITLE
(add) add tests to the new KindomService methods

### PR DESCRIPTION
**Pull request by pitner33**

### CODE REVIEW

**Comment by pitner33** [comment]
TRB-18 valahol lemaradt

**Review by bleaksmile — APPROVED** [review]

File: src/test/java/com/greenfox/tribes1/Kingdom/KingdomServiceTest.java [hunk 1]
```diff
@@ -17,7 +26,11 @@
   KingdomRepository kingdomRepository;
 
   private Kingdom validKingdom = new Kingdom("Narnia");
+  private Kingdom validKingdom2 = new Kingdom("Rueppellii");
```
Comment (csadesz) [inline src/test/java/com/greenfox/tribes1/Kingdom/KingdomServiceTest.java]: It is not a good practice to use variable names like this. Maybe you can refactor these names to validKingdomRueppellii, validKingdomNarnia

**Review by csadesz — APPROVED** [review]

**Review by csadesz — CHANGES_REQUESTED** [review]
variable's name

**Comment by pitner33** [comment]
variable names refactored

**Review by csadesz — APPROVED** [review]

**Pull request merged**